template.yaml

```diff
@@ -0,0 +1,41 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
What does this file do again?

Reply:
@nakedsushi This file allows you to test the function locally via `sam build` and `sam local start-api`